### PR TITLE
regex not working with Xcode 8. added regex to support both Xcode 8 a…

### PR DESCRIPTION
**Pull request by rafael-chavez**
…nd previous versions.

Added line for clearer error.

### CODE REVIEW

**Comment by rafael-chavez** [comment]
@imurchie @jlipps @Jonahss

File: lib/simctl.js [hunk 1]
```diff
@@ -157,12 +157,13 @@ async function getDevices (forSdk:string = null):Object {
       //   iPhone 4s
       //   A99FFFC3-8E19-4DCF-B585-7D9D46B4C16E
       //   Shutdown
-      let lineRe:RegExp = /^    ([^\(]+) \(([^\)]+)\) \(([^\)]+)\)/;
+      let lineRe:RegExp = /([^\s].+) \((\w+-.+\w+)\) \((\w+)\)/;
       let lineMatch:Object = lineRe.exec(line);
       if (lineMatch === null) {
-        throw new Error('Could not match line');
+        throw new Error('Could not match line: ' + line);
```
Comment (rafael-chavez) [inline lib/simctl.js]: do we prefer template literals here like such:

```
`Could not match line: ${line}`
```

Comment (jlipps) [inline lib/simctl.js]: yes we do, thank you

**Comment by jlipps** [comment]
LGTM, though admittedly I don't know the regex as well as @Jonahss

File: lib/simctl.js (hunk 1 above)
Comment (imurchie) [inline lib/simctl.js]: I've been adding a comment including a saved regexp test, with the regexp and a variety of examples of the different sorts of input text expected from different versions, for clarity. See, for instance, https://github.com/appium/appium-adb/blob/71643c324927a3bedf340937d189437c398271ff/lib/tools/apk-utils.js#L90

Comment (rafael-chavez) [inline lib/simctl.js]: Great idea @imurchie. Updated!

**Comment by rafael-chavez** [comment]
updated @jlipps @imurchie

**Comment by imurchie** [comment]
Awesome! This is great!

**Comment by imurchie** [comment]
Published in `node-simctl@3.4.3`.

**Pull request merged**